app/controllers/api/middleware.js

```diff
@@ -33,3 +33,6 @@ exports.files = async(function*(req, res) {
 exports.services = async(function*(req, res) {
   proxy_call(req, res, "/mw-api/services");
 });
+exports.service_announcement = async(function*(req, res) {
+  proxy_call(req, res, "/mw-api/service_announcement");
```

Comment:
Is this route used somewhere at the moment? In `js/middleware.js` I only see calls to `/api/mw/services` and `/api/mw/files`

Reply:
It is, there was some code missing in app/assets/js/middleware.js